addons/osfstorage/models.py

```diff
@@ -294,6 +294,9 @@ def remove_tag(self, tag, auth, save=True, log=True):
         if self.node.is_registration:
             # Can't perform edits on a registration
             raise NodeStateError
+        # TODO does modm let you do this???!
```

Comment:
No, I'm stupid. I was getting a `.values_list('name')` instead of `.values_list('name', flat=True)`.

Fixed.